xray/conventions.py

```diff
@@ -804,9 +804,13 @@ def stackable(dim):
         return True
 
     coord_names = set()
+    if isinstance(drop_variables, basestring):
+        drop_variables = set([drop_variables,])
```

Comment:
actually -- we should always convert drop_variables to a set. I was thinking something like:

```python
if isinstance(drop_variables, basestring):
    drop_variables = [drop_variables]
elif drop_variables is None:
   drop_variables = []
drop_variables = set(drop_variables)
```